tests/FakeItEasy.Tests/Configuration/AnyCallConfigurationTests.cs

```diff
@@ -50,6 +50,18 @@ public void WithReturnType_should_set_the_type_to_the_configured_rule()
         }
 
         [Test]
+        public void WithNonVoidReturnType_should_set_the_flag_to_apply_to_all_return_types()
+        {
+            // Arrange
+
+            // Act
+            this.configuration.WithNonVoidReturnType();
+
+            // Assert
+            Assert.That(this.callRule.ApplicableToAllNonVoidReturnTypes, Is.EqualTo(true));

```

Comment:
Well, it's also read in `AnyCallCallRule`... but yes, it's the only place outside the class where it's read. It doesn't really bother me, though; IMO, if we can set it, we should be able to get it too.